<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>payFine</class>
 <widget class="QDialog" name="payFine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>缴纳罚款</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(158, 216, 255);</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>缴纳罚款</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>缴纳钱数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>当前欠款：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>显示余额或存款</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>(&lt;0表示有余额)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>591</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
